### ISSUE TITLE
Traefik prevents media streaming

### ISSUE THREAD

**Issue by tscs37** [containous/traefik#1184]
### What version of Traefik are you using?
```
Version:      v1.2.0-rc1
Codename:     morbier
Go version:   go1.7.5
Built:        2017-02-06_08:55:03PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml...)?
```json
{
 "GraceTimeOut": 10,
 "Debug": false,
 "CheckNewVersion": true,
 "AccessLogsFile": "/var/log/traefik/access.log",
 "TraefikLogsFile": "/var/log/traefik/traefik.log",
 "LogLevel": "WARN",
 "EntryPoints": {
  "http": {
   "Network": "",
   "Address": ":80",
   "TLS": null,
   "Redirect": {
    "EntryPoint": "https",
    "Regex": "",
    "Replacement": ""
   },
   "Auth": null,
   "Compress": false
  },
  "https": {
   "Network": "",
   "Address": ":443",
   "TLS": {
    "MinVersion": "",
    "CipherSuites": null,
    "Certificates": [
     {
      "CertFile": "/etc/traefik/ssl/cert.pem",
      "KeyFile": "/etc/traefik/ssl/key.pem"
     }
    ],
    "ClientCAFiles": null
   },
   "Redirect": null,
   "Auth": null,
   "Compress": false
  }
 },
 "Cluster": null,
 "Constraints": [],
 "ACME": {
  "Email": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
  "Domains": [
   {
    "Main": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
    "SANs": null
   }
  ],
  "Storage": "/etc/traefik/acme/acme.json",
  "StorageFile": "",
  "OnDemand": false,
  "OnHostRule": false,
  "CAServer": "",
  "EntryPoint": "https",
  "DNSProvider": "",
  "DelayDontCheckDNS": 0,
  "ACMELogging": false,
  "TLSConfig": null
 },
 "DefaultEntryPoints": [
  "http",
  "https"
 ],
 "ProvidersThrottleDuration": 2000000000,
 "MaxIdleConnsPerHost": 100,
 "InsecureSkipVerify": true,
 "Retry": {
  "Attempts": 3
 },
 "Docker": null,
 "File": {
  "Watch": true,
  "Filename": "/etc/traefik/rules.toml",
  "Constraints": null
 },
 "Web": {
  "Address": ":8080",
  "CertFile": "",
  "KeyFile": "",
  "ReadOnly": true,
  "Statistics": null,
  "Metrics": null,
  "Auth": null
 },
 "Marathon": null,
 "Consul": null,
 "ConsulCatalog": null,
 "Etcd": null,
 "Zookeeper": null,
 "Boltdb": null,
 "Kubernetes": null,
 "Mesos": null,
 "Eureka": null,
 "ECS": null,
 "Rancher": null,
 "ConfigFile": "/etc/traefik/config.toml"
}
```

### What did you do?

I attempted to access a Zoneminder Stream through Traefik which was configured to use auto+mpeg streaming mode (autodetecting if streaming is available and if yes, using MPEG)

### What did you expect to see?

I expected to see the live stream of the camera in the preview window.

### What did you see instead?

The preview window is empty and after some time traefik logs the follwoing:

```
time="2017-02-23T13:17:59+01:00" level=warning msg="Error copying upstream response Body: net/http: request canceled"
```

The request keeps being marked as pending in the browser, when closing the browser the request will be logged as cancelled, like noted above.

**Comment by emilevauge** [comment]
Hi @tscs37, could help us reproducing this issue? Maybe with a docker-compose file or something similar?

**Comment by tscs37** [comment]
I've attempted to reproduce the problem using a docker-compose file but I've been unable to, however I noticed that ZoneMinder uses the header

 `Content-Type:multipart/x-mixed-replace;boundary=ZoneMinderFrame`

which the docker-compose variant does not, it uses 

`Content-Type:application/x-shockwave-flash`

I guess the former header causes traefik to wait for the end of the response?

I also checked against directly accessing the backend via IP, which opens the console and allows for normal streaming.

**Comment by timoreimann** [comment]
Chances are this will get fixed by containous/oxy#15.

**Comment by tscs37** [comment]
The newest update seems to have fixed this, many thanks.
